### PR TITLE
Import script for St Helens (closes #2124)

### LINKED ISSUE #2124: Import E08000013-St Helens Metropolitan Borough Council
Xpress DC

### PR DESCRIPTION
**Pull request by GeoWill**
https://trello.com/c/5HLxjs3v

### CODE REVIEW

**Comment by chris48s** [comment]
Had a quick look over this one and there's quite a few obviously misplaced addresses in pickers (which are then missing from the postcodes they're supposed to be assigned to)

examples (but not necessarily limited to):

79 Newlove Avenue
WA119TB

32 Cunningham Court
WA101JT

6 Alexander Court
WA95GG

22 Dorothy Street
WA92RL

107A Springfield Road
WA93RR

Suspect some of these could also be errors in addressbase.
Any chance you can have a look into these? If the simplest thing is to bin data rather than fix it, do.
If there addressbase errors we need to feed back to OS, can you bung them in the spreadsheet. Ta

File: polling_stations/apps/data_collection/management/commands/import_st_helens.py [hunk 1]
```diff
@@ -3,44 +3,45 @@
...
-            rec["accept_suggestion"] = True
+        if uprn == "39086246":
+            return None
+
+        # UPRNs 39089002 through 39089022 look fishy. Not sure if it's council or addressbase.
+        if "Alexander Court" in record.addressline1:
+            rec["uprn"] = ""
+
+        if uprn == "39091880":  # 79 Newlove Avenue
+            rec["postcode"] = "WA104DS"
```
Comment (chris48s) [inline polling_stations/apps/data_collection/management/commands/import_st_helens.py]: This doesn't actually fix this one.
You might need to set `rec["accept_suggestion"] = False` here too

File: polling_stations/apps/data_collection/management/commands/import_st_helens.py [hunk 2]
```diff
@@ -3,44 +3,45 @@
...
+        if uprn == "39091918":  # 32 Cunningham Court
+            rec["postcode"] = "WA104GA"
+        if uprn == "39099008":  #
+            rec["postcode"] = "WA95RL"
+        if uprn == "39097964":  # 107A Springfield Road
+            rec["postcode"] = "WA103RR"
```
Comment (chris48s) [inline polling_stations/apps/data_collection/management/commands/import_st_helens.py]: Also, there's definitely a few more of these to spot e.g:

WA102TL
14A Hartford Road

WA102TJ
53 The Boulevard

File: polling_stations/apps/data_collection/management/commands/import_st_helens.py (hunk 1 above)
Comment (GeoWill) [reply polling_stations/apps/data_collection/management/commands/import_st_helens.py]: I only checked the fixed postcode :roll_eyes:

**Comment by GeoWill** [comment]
Rebased and squashed, hope there's not more - if there are let me know how you're finding them!

**Comment by chris48s** [comment]
don't think your rebase went to plan :grimacing: 
if its easier to copy the script to a new branch and create a new PR for it, go for it..

**Pull request merged**